garrysmod/gamemodes/terrortown/entities/entities/ttt_confgrenade_proj.lua

```diff
@@ -39,7 +39,7 @@ local function PushPullRadius(pos, pusher)
 
             target:SetVelocity(vel)
 
-            target.was_pushed = {att=pusher, t=CurTime()}
+            target.was_pushed = {att=pusher, t=CurTime(), wep="ttt_weapon_confgrenade"}

```

Comment:
That's what I get for retyping the change instead of copying it. I'll fix it momentarily.